### DISCUSSION TITLE
Is there a future release planned for relying upon Bouncy Castle Provider (FIPS Distribution) instead of bc-prov?

### DISCUSSION THREAD

**Discussion by randheersingh437** [webauthn4j/webauthn4j#421] [Q&A] [answered]
Is there a future release planned for relying upon Bouncy Castle Provider (FIPS Distribution) instead of bc-prov?
https://mvnrepository.com/artifact/org.bouncycastle/bc-fips

**Comment by ynojima** [comment, marked as answer]
I suppose you are requesting because webauthn4j-util depended on bcprov-jdk15on and bcpkix-jdk15on, but it was my mistake.
Actually, bcprov-jdk15on and bcpkix-jdk15on are only used by webauthn4j-test, which is test libarary and I assume you are not using it.
I removed the implementation dependency. https://github.com/webauthn4j/webauthn4j/pull/423
Hope this helps.

**Reply by randheersingh437** [reply]
Thank you so very much for taking care of this. Do we have any tentative schedule for releasing this change in 0.14.1 or in 0.15.0 release? We are in evaluating this library for our FIDO implementation.

**Reply by ynojima** [reply]
I released [0.14.1](https://github.com/webauthn4j/webauthn4j/releases/tag/0.14.1.RELEASE) now. Please check it!
